docs/start/setup-channel.mdx

```diff
@@ -165,7 +165,72 @@ print(result)
     from qiskit_ibm_runtime import QiskitRuntimeService
 
     service = QiskitRuntimeService()
-    ```
+  ```
+
+## Set up account using REST APIs
```

Comment:
I'm not super familiar with how the rest api works, but I think maybe this section should be a sub-section under the "set up using IBM Quantum Platform" heading? You are still accessing the systems through IQP, not IBM cloud right? Also would be good for this section to follow the same structure of steps as the other instructions in this page for consistency